docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
```

Comment:
what the verion there? the one for the compose file?

Reply:
yes. They have that all over in their docs.